README.md

```diff
@@ -79,6 +80,7 @@ Open Source Audio Apps
 | FlexASIO | [dechamps/FlexASIO](https://github.com/dechamps/FlexASIO) | Universal ASIO driver for Windows |
 | [Guitarix](https://guitarix.org/) | [SourceForge → guitarix](https://sourceforge.net/projects/guitarix/) | GNU/Linux Virtual Amplifier |
 | [Helio Workstation](https://helio.fm/) | [helio-fm/helio-workstation](https://github.com/helio-fm/helio-workstation) | Sequencer |
+| [NewMixer](https://github.com/jatinchowdhury18/NewMixer) | [jatinchowdhury18/NewMixer](https://github.com/jatinchowdhury18/NewMixer) | Mixing Software |
```

Comment:
Can you move this to the apps section, the plugins are supposed to be items that embed in a host app. Thanks!

Reply:
Hi Chris,

I believe this is in the apps section, no? (picture from my fork attached)
![OpenAudio](https://user-images.githubusercontent.com/13724188/58139684-b759e480-7bf0-11e9-9a65-e5b2bb033f93.PNG)

Reply:
My mistake, was reading it on my phone!